test/protobuf/decoder_test.exs

```diff
@@ -126,7 +126,7 @@ defmodule Protobuf.Decoder.Test do
       use Protobuf, from: Path.expand("../proto/mumble.proto", __DIR__)
     end
 
-    msg = MumbleProto.Authenticate.new(username: "bitwalker")
+    msg = MumbleProto.MumbleProto.Authenticate.new(username: "bitwalker")
```

Comment:
While I agree this should be the expected behaviour, it's definitely worth pointing out that this is a breaking change. If other people are using this library they may well need to update their code or proto files to incorporate this change. Also best that @bitwalker is aware that the new default behaviour is that package namespaces will be honoured and could break any existing code that doesn't expect this.